Bottom: c52a2af0f8b9c3237bfea83ef532cb6e029b447a
Top:    e3d9cedddd4a15e3608768a4c70b09e8d1a41bc4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 15:52:30 +0100

Refresh of generate-pcb-menu-res-and-gpcb

---

diff --git a/po/POTFILES.in b/po/POTFILES.in
index 0e5e331..8d62595 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -16,7 +16,7 @@ src/edif.c
 src/error.c
 src/file.c
 src/find.c
-src/gpcb-menu.res
+src/gpcb-menu.res.in
 src/hid/batch/batch.c
 src/hid/common/actions.c
 src/hid/common/hidnogui.c
@@ -40,7 +40,7 @@ src/misc.c
 src/move.c
 src/netlist.c
 src/parse_y.c
-src/pcb-menu.res
+src/pcb-menu.res.in
 src/polygon.c
 src/rats.c
 src/report.c
